helm/fdb-operator/templates/rbac/rbac_role.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think it's better to use a role rather than a  cluster role, to support using the operator in environments where the person running the operator doesn't have cluster-wide access.

Reply:
I agree, we should have `role` in place of ClusterRole here.